Студентам для решения домашнего задания/case_1.py

```diff
@@ -0,0 +1,59 @@
+import os, glob, csv, re, chardet
+
+path = 'C:\\Users\\Администратор\\Desktop\\Студентам для решения домашнего задания'
+file_for_write = 'C:\\Users\\Администратор\\Desktop\\Студентам для решения домашнего задания\\main_data.txt'

```

Comment:
Specifying absolute paths in code is completely unacceptable!
It would be more correct to specify a relative path: 
```path = os.path.getcwd()```